ads/colombia.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateDataExists} from '../src/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function colombia(global, data) {
+  validateDataExists(data, [
+    'clmb_slot', 'clmb_position', 'clmb_section',
+    'clmb_divid', 'loadingStrategy',
+  ]);
+  global.clmbSlot = data.clmbSlot;
+  global.clmbPosition = data.clmbPosition;
+  global.clmbSection = data.clmbSection;
+  global.clmbDivid = data.clmbDivid;
+  global.context.observeIntersection(function(d) {
+    d.forEach(function() {
+      ampPushNotify.itemTrackURL(global);
```

Comment:
Makes sense. Please be explicit that it is a global variable by dereferencing it from `global`.

There is a race condition here. You cannot know that the intersection callback won't come after your script loads. This would be easily dealt with by starting the observation in the callback of the script load.